model-metadata/epiENGAGE-lop_norm.yml

```diff
@@ -0,0 +1,36 @@
+team_name: "epiENGAGE"
+team_abbr: "epiENGAGE"
+model_name: "Ensemble models using flu-metrocast forecast submissions"
+model_abbr: "lop_norm"
+model_version: "1.0"
+model_contributors: [
+  {
+    "name": "Dongah Kim",
+    "affiliation": "Department of Integrative Biology, University of Texas Austin",
+    "email": "dongah.kim@austin.utexas.edu"
+  },
+  {
+    "name": "Li Shandross",
+    "affiliation": "University of Massachusetts Amherst",
+    "email": "lshandross@umass.edu"
+  },
+    {
+    "name": "Evan Ray",
+    "affiliation": "University of Massachusetts Amherst",
+    "email": "elray@umass.edu"
+  },
+  {
+    "name": "Nick G. Reich",
+    "affiliation": "Department of Biostatistics and Epidemiology, University of Massachussetts Amherst",
+    "email": "nick@umass.edu"
+  },
```

Comment:
I don't think we should include Evan and Li here, since they didn't contribute to this model itself. They did contribute to the package that is helping you run the model, but not to this particular model, so I would leave them out.